dist/src/main/dist/conf/simulator.properties

```diff
@@ -171,7 +171,7 @@ MACHINE_SPEC=hardwareId=c3.xlarge,imageId=us-east-1/ami-1b3b2472
 #                                   git=myRepository/myBranch - to build a version from a branch in a specific repository.
 #   Use GIT_CUSTOM_REPOSITORIES to specify custom repositories. The main Hazelcast repository is always named "origin".
 #
-HAZELCAST_VERSION_SPEC=maven=3.4.2
+HAZELCAST_VERSION_SPEC=maven=3.5-SNAPSHOT
```

Comment:
you are right. 
outofthebox is fine here, assuming Jenkins has it's own property explicitly set.